dev/releases/utils.py

```diff
@@ -155,10 +158,17 @@ def check_whether_github_release_exists(tag):
 def initialize_github(token=None):
     global GITHUB_INSTANCE, CURRENT_REPO
     if GITHUB_INSTANCE != None or CURRENT_REPO != None:
-        error("Global variables GITHUB_INSTANCE and CURRENT_REPO "
+        error("Global variables GITHUB_INSTANCE and CURRENT_REPO"
               + " are already initialized.")
     if token == None and "GITHUB_TOKEN" in os.environ:
         token = os.environ["GITHUB_TOKEN"]
+    if token == None:
+        temp = subprocess.run(["git", "config", "--get", "github.token"], text=True, capture_output=True)
+        if temp.returncode == 0:
+            token = temp.stdout.strip()
+    if token == None and os.path.isfile(os.path.expanduser('~') + '/.github_shell_token'):
+        with open(os.path.expanduser('~') + '/.github_shell_token', 'r') as token_file:
+            token = token_file.read().replace('\n', '')
```

Comment:
Note to self, this could be:
```suggestion
            token = token_file.read().strip()
```